src/styles.scss

```diff
@@ -5,7 +5,7 @@
 @include igx-theme($default-palette, $legacy-support: true);
 
 :root {
-    @include css-vars-from-palette($default-palette);
+    @include igx-palette-vars($default-palette);
```

Comment:
This is no longer needed as the palette colors are included as CSS variables by default now.